components/devtools/actors/root.rs

```diff
@@ -34,7 +34,7 @@ enum AddonMsg {}
 struct ListTabsReply {
     from: String,
     selected: u32,
-    tabs: Vec<TabActorMsg>,
+    tabs: Vec<BrowsingContextActorMsg>,
```

Comment:
at the moment, we still use tabs to refer to this collection in the Devtools codebase. if this changes I will update it here as well.